python/pyspark/ml/tests.py

```diff
@@ -1655,6 +1655,25 @@ def test_multinomial_logistic_regression_with_bound(self):
             np.allclose(model.interceptVector.toArray(), [-0.9057, -1.1392, -0.0033], atol=1E-4))
 
 
+class MultilayerPerceptronClassifierTest(SparkSessionTestCase):
+
+    def test_multilayer_perceptron_classifier(self):
+
+        data_path = "data/mllib/sample_multiclass_classification_data.txt"
```

Comment:
I prefer data set generated in code. Like other test cases.

Reply:
For other algorithms, I agree with you. However, too simple dataset is not ideal to test MLPC. This dataset is a tiny dataset with 150 rows and 4 columns, so I think we can use it. Thanks.